common/shlibs

```diff
@@ -3990,3 +3990,5 @@ liburing.so.1 liburing-0.7_1
 libbson-1.0.so.0 libbson-1.17.0_1
 libsonic.so.0 libsonic-0.2.0_1
 libtickit.so.3 libtickit-0.4.1_1
+libsimavr.so.1 simavr-1.6_2
```

Comment:
please incorporate changes to common/shlibs into the commit that adds the package

Reply:
done